### ISSUE TITLE
WordPress Importer: Remove outdated help modal

### ISSUE THREAD

**Issue by dezzie** [Automattic/wp-calypso#33386]
#### Steps to reproduce
1. Starting at URL: https://wordpress.com/settings/import/
2. Choose on the WordPress importer
3. Go to the "`exporting your content`" link

#### What I expected
The latest up-to-date documentation for this tool, with up-to-date images and info from the latest version of WordPress.com.

#### What happened instead
- A modal with outdated help content, supported features, and screenshots.
- (As a user I don't know this, nor do I know that it's a selected chunk of information from a broader support article.)

#### Browser / OS version
- everywhere

#### Screenshot / Video
<img width="1297" alt="trash 5-28 - 2" src="https://user-images.githubusercontent.com/1301932/58499883-9d675700-8146-11e9-8eaf-4f01bc23f059.png">

#### Context / Source
Help and support information should fall into consistent categories and types for users. Short and sweet information can go in the UI copy, or a tooltip. Anything long-form should belong in the support docs. Modals are best used in situations where you need users to stop paying attention to anything else except for what is in the modal, such as displaying an important message that needs action, or get them to complete a specific task. In a troubleshooting workflow users need to be able to go back and forth between their work and docs.

Not only is en.support the primary place that users are sent to when they have a problem, it's much easier and faster to keep up to date, and runs no risk of introducing one-off accessibility issues or other tech debt. We don't have to manage a short vs full version of documentation between two places, as we do here. No need to create Github issues and wait on reviews, plus a wider group of people do the work themselves.

#fix-the-flows, #importers-design, #muriel

**Comment by dezzie** [comment]
### Delete:

- [ ] The entire modal situation

### Update:

- [ ] Change existing `exporting your content` link to `https://en.support.wordpress.com/coming-from-self-hosted/`
- [ ] Review en.support documentation to make sure it's up-to-date both ways.
